tests/unit/core/reducers/test_user.js

```diff
@@ -0,0 +1,42 @@
+import { logOutUser } from 'core/actions';
+import reducer, {
+  userProfileLoaded,
+} from 'core/reducers/user';
+import { createUserProfileResponse } from 'tests/unit/helpers';
+
+describe(__filename, () => {
+  describe('reducer', () => {
+    it('initializes properly', () => {
+      const { id, username } = reducer(undefined);
+      expect(id).toBe(null);
+      expect(username).toEqual(null);
+    });
+
+    it('ignores unrelated actions', () => {
+      const state = { id: 12345, username: 'john' };
```

Comment:
Could this use the `initialiState` from the actual reducer instead? I just like to stick with realistic data.

Reply:
That would mean exposing the `initialState`?

Reply:
Yeah, [we do that in a fair number of reducers](https://github.com/mozilla/addons-frontend/blob/master/src/amo/reducers/featured.js#L4), I think it's fine.